docs/src/main/sphinx/installation/cli.rst

```diff
@@ -48,15 +48,59 @@ recognizes these certificates.
 
 Authentication
 --------------
+Password/LDAP Authentication
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 You can override your username with the ``--user`` option. It defaults to your
 operating system username. If your Trino server requires password
 authentication, use the ``--password`` option to have the CLI prompt for a
 password. You can set the ``TRINO_PASSWORD`` environment variable with the
 password value to avoid the prompt.
 
+When Trino is configured with LDAP authentication, invoking CLI requires additional options
```

Comment:
@mosabua I have tried to simplify this sentence from the earlier structure. I thought the previous sentence was a little bit long and complex to quickly understand. Let me know your thoughts on this